src/consts.ts

```diff
@@ -1,6 +1,14 @@
 import { FlickingOptions, EventType, Direction, AxesEventType, StateType, MoveTypeSnapOption, MoveTypeFreeScrollOption } from "./types";
 import { checkTranslateSupport } from "./utils";
 
+export const MOVE_TYPE: {
+  SNAP: "snap";
+  FREE_SCROLL: "freeScroll";
+} = {
+  SNAP: "snap",
+  FREE_SCROLL: "freeScroll",
+};
```

Comment:
@WoodNeck What is this syntax?

Reply:
By doing that, `MOVE_TYPE.SNAP`'s type inference will be `"snap"`, while `string` will be inferred if not.